### PR TITLE
Awesome Books

### PR DESCRIPTION
**Pull request by RiyaBulia12**
This PR includes the following:
- Created a collection that keeps a list of books using an array of objects
- Added a function to add a new book to the collection, with title and author.
- Added a function to remove a book from the collection
- Displayed all books saved in the collection in the top part of the page.
- When a user clicks the "Add" button:
   - A new book is added to the collection.
   - The new book is displayed on the page.
- When a user clicks the "Remove" button:
   - The correct book is removed from the collection.
   - The correct book disappears from the page.
- Data is preserved in the browser's memory by using localStorage

### Coding Partner
- Ehma  @ehma90 

## ScreenShot
Screenshot 2022-08-02 at 2 28 05 PM

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ getBooksList();
 function getBooksList() {
 
    if (getBooks.length) {
-      window.addEventListener('DOMContentLoaded', (event) => {
+      window.addEventListener('DOMContentLoaded', () => {
```
Comment (Kingstalux) [inline script.js]: - [ ] Good job so far :+1: Kindly fix this console error on your site. This error prevents the adding of a book, therefore kindly clear your local storage and rectify this issue :100: 
- Console error:
Screenshot from 2022-08-02 10-12-04

File: script.js [hunk 2]
```diff
@@ -61,11 +54,29 @@ function populateBookDetails(item) {
    const removeBtn = bkRow.appendChild(document.createElement('button'));
    removeBtn.innerHTML = 'Remove';
    removeBtn.setAttribute('class', 'removeBook')
```
Comment (Kingstalux) [inline script.js]: - [ ] Kindly make sure that when the remove button is clicked, the correct element is being deleted from your site and from the local storage :smile:

File: script.js [hunk 3]
```diff
@@ -1,77 +1,69 @@
...
+   this.id = id;
+   this.title = title;
+   this.author = author;
+}
+const bookList = [];
+const addBookBtn = document.getElementById('addBookBtn');
+const bookTable = document.getElementById('bookTable');
+const emptyMessage = document.querySelector('.emptyMessage');
 
+const getBooks = JSON.parse(localStorage.getItem('books'));
```
Comment (Kingstalux) [inline script.js:10]: - [ ] Kindly make sure your data is being preserved in your browser and as a result, if your site is manually refreshed, the correct list of books will still be displayed :+1:

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @RiyaBulia12 and @ehma90 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :clap: 
:white_check_mark: Good commit history :+1: 
:white_check_mark: Linters are passing :ok_hand: 
:white_check_mark: Professional README file :100: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @RiyaBulia12 and @ehma90 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- Professional README.md file 💯


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




## Optional suggestions

- [OPTIONAL] I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**